google/cloud/forseti/scanner/scanners/config_validator_util/cv_data_converter.py

```diff
@@ -111,6 +111,8 @@ def convert_data_to_cv_asset(resource, data_type):
     ancestry_path = generate_ancestry_path(resource.full_name)
 
     data = json.loads(resource.data)
+    if isinstance(data, list):
```

Comment:
This seems ok, but I'm curious about how/why. How is this error reproduced? Is it for certain resource types? What error did you see?

Reply:
I can reproduce this error by just running `forseti scanner run` for Config Validator scanner.

It looks like I'm getting these errors due to CRM Org policy, which looks like:
`[{'boolean_policy': {'enforced': True}, 'constraint': 'constraints/compute.disableSerialPortAccess', 'update_time': {'nanos': 215000000, 'seconds': 1539891911}}]`

The resources that are pulled in are all of type dict, while this is a list of dicts, which causes the `cleanup_dict` function [here](https://github.com/forseti-security/forseti-security/blob/7bd5107490c8e099dba45a5de9581c5e9f95e24b/google/cloud/forseti/scanner/scanners/config_validator_util/cv_data_converter.py#L151) to error out, since it is expecting a dict.

Here's the error I'm getting when running Config Validator scanner:

```
2020-01-24 09:26:26,765 ERROR google.cloud.forseti.scanner.scanner(run): Error running scanner: ConfigValidatorScanner: 'Traceback (most recent call last):
  File "/Users/user-xx/.virtualenvs/forseti/lib/python3.7/site-packages/google/cloud/forseti/scanner/scanner.py", line 119, in run
    scanner.run()
  File "/Users/user-xx/.virtualenvs/forseti/lib/python3.7/site-packages/google/cloud/forseti/scanner/scanners/config_validator_scanner.py", line 193, in run
    for flattened_violations in self._retrieve_flattened_violations():
  File "/Users/user-xx/.virtualenvs/forseti/lib/python3.7/site-packages/google/cloud/forseti/scanner/scanners/config_validator_scanner.py", line 174, in _retrieve_flattened_violations
    for violations in self.validator_client.paged_review(cv_assets):
  File "/Users/user-xx/.virtualenvs/forseti/lib/python3.7/site-packages/google/cloud/forseti/scanner/scanners/config_validator_util/validator_client.py", line 106, in paged_review
    for asset in assets:
  File "/Users/user-xx/.virtualenvs/forseti/lib/python3.7/site-packages/google/cloud/forset
Traceback (most recent call last):
  File "/Users/user-xx/.virtualenvs/forseti/lib/python3.7/site-packages/google/cloud/forseti/scanner/scanner.py", line 119, in run
    scanner.run()
  File "/Users/user-xx/.virtualenvs/forseti/lib/python3.7/site-packages/google/cloud/forseti/scanner/scanners/config_validator_scanner.py", line 193, in run
    for flattened_violations in self._retrieve_flattened_violations():
  File "/Users/user-xx/.virtualenvs/forseti/lib/python3.7/site-packages/google/cloud/forseti/scanner/scanners/config_validator_scanner.py", line 174, in _retrieve_flattened_violations
    for violations in self.validator_client.paged_review(cv_assets):
  File "/Users/user-xx/.virtualenvs/forseti/lib/python3.7/site-packages/google/cloud/forseti/scanner/scanners/config_validator_util/validator_client.py", line 106, in paged_review
    for asset in assets:
  File "/Users/user-xx/.virtualenvs/forseti/lib/python3.7/site-packages/google/cloud/forseti/scanner/scanners/config_validator_scanner.py", line 152, in _retrieve
    resource, data_type)
  File "/Users/user-xx/.virtualenvs/forseti/lib/python3.7/site-packages/google/cloud/forseti/scanner/scanners/config_validator_util/cv_data_converter.py", line 116, in convert_data_to_cv_asset
    cleanup_dict(data)
  File "/Users/user-xx/.virtualenvs/forseti/lib/python3.7/site-packages/google/cloud/forseti/scanner/scanners/config_validator_util/cv_data_converter.py", line 159, in cleanup_dict
    for key, value in list(raw_dict.items()):
AttributeError: 'list' object has no attribute 'items'
```

Let me know if you need any more clarification.

Reply:
I believe this issue has been resolved specifically for Org Policy with a PR that I hope to get merged in today. https://github.com/forseti-security/forseti-security/pull/3571/files

Reply:
Ok cool, looks like it is utilizing data[0] as well there. Will close this PR once that one gets merged in. Thanks!